tests/publiccloud/run_ltp.pm

```diff
@@ -170,13 +170,16 @@ sub cleanup {
 
     # Ensure that the ltp script gets killed
     type_string('', terminate_with => 'ETX');
+    record_info('LTP Logs', 'upload');
     $self->upload_ltp_logs();
-
+    record_info('LTP folders', 'upload');
     if ($self->{my_instance} && script_run("test -f $root_dir/log_instance.sh") == 0) {
         assert_script_run($root_dir . '/log_instance.sh stop ' . $self->instance_log_args());
         assert_script_run("(cd /tmp/log_instance && tar -zcf $root_dir/instance_log.tar.gz *)");
         upload_logs("$root_dir/instance_log.tar.gz", failok => 1);
     }
+    assert_script_run("(cd /tmp/runltp.root && tar -zcf $root_dir/runltproot.tar.gz *)");
```

Comment:
from where this path is coming ? ( I mean "/tmp/runltp.root" )  
@acerv  please check if that is what you wanted

Reply:
I'm not sure you need the whole runltp-ng temporary folder. `debug.log` is enough for you.
The folder is created by runltp-ng and it can be also changed via `--tmp-dir` parameter.

Reply:
so maybe better to explicitly set something during invocation of runltp-ng  so we will be protected from any changes in default path ?

Reply:
Yes, probably it's better to force a specific runltp-ng temporary folder (maybe in the working directory).

Reply:
@m-dati  please tweak runltp-ng to use `--tmp-dir` and than copy only debug.log from there

Reply:
Already prepared a new version, simply using the predefined filename `/tmp/runltp.\$USER/latest/debug.log` to upload. Will soon update this PR

Reply:
but if instead here a defined tmp-dir is preferred, I could change the runltp-ng command line adding that parameter

Reply:
Using` --tmp-dir ` we change the default **/tmp** dir with another different **tmp-root** folder, but the appended _LTP sub-structure_ remains the same. I.e. after a test using ` --tmp-dir = "/root/runltp_logs"`, we note we still have the debug file in the inner path `runltp.$USER/latest/debug.log` in the tree:
```
# wait_serial expected: qr/j4PDP-\d+-/u
# Result:

/root/runltp_logs:
total 4
drwxr-xr-x 3 root root 4096 Dec  9 09:13 runltp.root

/root/runltp_logs/runltp.root:
total 4
lrwxrwxrwx 1 root root   41 Dec  9 09:13 latest -> /root/runltp_logs/runltp.root/tmp8c0quowq
drwx------ 3 root root 4096 Dec  9 09:13 tmp8c0quowq

/root/runltp_logs/runltp.root/tmp8c0quowq:
total 600
-rw-r--r-- 1 root root 570742 Dec  9 09:13 debug.log
-rw-r--r-- 1 root root   1256 Dec  9 09:13 results.json
drwxr-xr-x 2 root root   4096 Dec  9 09:13 runtest
-rw-r--r-- 1 root root  31784 Dec  9 09:13 syscalls

/root/runltp_logs/runltp.root/tmp8c0quowq/runtest:
total 32
-rw-r--r-- 1 root root 31784 Dec  9 09:13 syscalls
No debug log
j4PDP-0-
```

Therefore, avoiding to add --tmp-dir in the runltp-ng command line, using the _default tmp-dir_ `/tmp`, the full log structure is better standardized and reusable, having our log name: `/tmp/runltp.$USER/latest/debug.log`